simple/getcreds.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
@NishikaDeSilva Add license header.

Reply:
Please refer to comment in https://github.com/wso2/kubernetes-apim/pull/162/files/79a955dad89701fd2ef9e1ec0278217422429dd6#r266556306